uri/patterns.yml

```diff
@@ -90,3 +90,20 @@ patterns:
       - name: non-english.txt
         start_offset: 19
         end_offset: 25
+
+
+  - name: GitHub Container Registry typos
```

Comment:
@aegilops Can you run the Markdown generator so this is added in the README?

Reply:
On it

Reply:
- Merged in changes from main
- generated README.md for everything